.gitignore

```diff
@@ -16,3 +16,4 @@
 coverage.txt
 codegpt
 dist
+.idea/*
```

Comment:
I'm sorry, but the code patch you provided is incomplete and doesn't contain any executable code. Please provide the full code that needs to be reviewed.